lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6893,7 +6987,14 @@ void EmitMethodFld(ParseNode *pnode, Js::RegSlot callObjLocation, Js::PropertyId
     bool isScoped = (byteCodeGenerator->GetFlags() & fscrEval) != 0 ||
         (isRoot && callObjLocation != ByteCodeGenerator::RootObjectRegister);
 
-    EmitMethodFld(isRoot, isScoped, pnode->location, callObjLocation, propertyId, byteCodeGenerator, funcInfo, registerCacheIdForCall);
```

Comment:
I wonder if this is the right place. Can you tell me how we got to EmitMethodFld with a symbol that shouldn't be loaded as an object property?